GameServerLib/API/ApiFunctionManager.cs

```diff
@@ -74,25 +120,52 @@ public static void SetGameObjectVisibility(GameObject gameObject, bool visibilit
             }
         }
 
+        /// <summary>
+        /// Gets the possible teams.
+        /// </summary>
+        /// <returns>Usually BLUE/PURPLE/NEUTRAL.</returns>
         public static List<TeamId> GetTeams()
         {
             return _game.ObjectManager.Teams;
         }
 
+        /// <summary>
+        /// Teleports an AI unit to the specified coordinates.
+        /// Instant.
+        /// TODO: Change to GameObjects.
+        /// </summary>
+        /// <param name="unit">AI unit to teleport.</param>
+        /// <param name="x">X coordinate.</param>
+        /// <param name="y">Y coordinate.</param>
         public static void TeleportTo(IObjAiBase unit, float x, float y)
         {
-            var coords = new Vector2(x, y);
-            var truePos = _game.Map.NavigationGrid.GetClosestTerrainExit(coords);
-
             CancelDash(unit);
-            unit.TeleportTo(truePos.X, truePos.Y);
+            unit.TeleportTo(x, y);
```

Comment:
What happens if you teleport into a wall?

Reply:
This calls ObjAiBase.TeleportTo, which already gets the closest terrain exit, I just removed it because it would be a useless extra call.